gherkin/ruby/lib/gherkin/query.rb

```diff
@@ -27,6 +27,9 @@ def update_feature(feature)
     end
 
     def update_rule(rule)
+      return if rule.nil?
```

Comment:
Copied this from the method above. I'm not sure this can happen (Same for features), but I'm not really sure how the parser generates the original AST which is then iterated through when we click "Run scenario" so I'll just copy and let it all work.

Reply:
It looks safer that way.
Good to me, even if useless. It may be useless for now, but that can change.